builders/base/builder/builder.toml

```diff
@@ -0,0 +1,16 @@
+[[buildpacks]]
```

Comment:
Good point. I'll go ahead and add in some gitignores to ignore the actual builder.toml files, since you're right that theyd get generated whenever someone runs the scripts locally